docs/docs/build/releases-and-apitags.mdx

```diff
@@ -32,7 +32,7 @@ There are no guarantees for API breakages or Fluid document compatibility among
 These are done on demand to preview new APIs or try fixes for our partners. They are represented by the `dev` tag in npm
 There are no guarantees for API breakages or Fluid document compatibility among the release changes. Assume you will have to throw away containers generated by older release candidates
 
-## API Support Levels {#api-support-levels}
+## API Support Levels \{#api-support-levels}
```

Comment:
See updated notes in the docs README. Opening brace must be escaped to prevent the contents from being treated as JSX syntax.